content/events/2021-online-unconference/_index.md

```diff
@@ -9,77 +9,77 @@ description = "Nordic RSE – Research Software Engineers in the nordics – Unc
 discussion topic, talk, demonstration, or any other type of program you would
-like to run.
+like to run beforehand (instructions below). But we will also collect on-site suggestions for contributions. No need to plan ahead! 
 
+To kick off the event on 29th we also have invited a few interesting talks. Stay tuned for updates!
 
-## Format and goal and scope
+## Format and scope
 
-We will run an informal unconference to bring the community of research software
-engineers together and to exchange ideas, solutions and challenges.
-In the unconference format **we encourage participants to shape the program**.
-We will provide support and infrastructure for that.
+We would like this event to be an informal space for exchanging ideas and experiences, learning something new and networking with people of the same interest group. You do not have to be a Research Software Engineer or a Researcher or a Software Engineer nor do you have to be in or be related to the Nordics. Everyone is welcome and encouraged to participate and shape the event to what you would like it to be!
 
-This event is not only limited to the Nordics and not only limited to participants
-who are or identify as RSEs, but anybody interested in the interface of research, software, and data is welcome.
+The nordicRSE team will provide support and infrastructure and give a short overview of the nordicRSE activities.
 
 
-## What contributions we welcome
+## Contributing
 
-We welcome *any kind of contribution* which respects the [Code of Conduct](./code-of-conduct).
-We will try to arrange a program based on your input.
+As an informal event, we welcome *any kind of contribution* which respects the [Code of Conduct](./code-of-conduct).
+If you have an idea you would like to discuss for the event, we encourage you to submit an issue to [this repository](https://github.com/nordic-rse/nordic-rse.github.io) to discuss with the community and secure a place in the program. 
+No idea yet? - No problem, we will also accept last minute contributions during the event. 
 
-Please also have a look at our [time length suggestions](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md).
+Please also have a look at our [time slot suggestions](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md).
 
-Here are some examples of possible contributions:
+Here are some examples to spark ideas:
 - *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
 - *Demonstrations:* show us some tools or software that you like
 - *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
 - *Talks*: teach us something new
 - *On-going projects*: share with others something unfinished and almost working that you would like to get feedback on
```

Comment:
I would add code e.g. "share with others unfinished codes..."

Reply:
is it really just codes?

Reply:
I like the idea of not focusing too much on just codes, as it seems to lead to self-imposed gate keeping: one of the people I encouraged to contribute felt that they did not fit in, because they don't work with things as "advanced" as the software/code development team, but rather "just simple" scripting, algorithms, and other things. A lot of our important RSE-related work is related to other things than pure coding. I know that this is just one case, but I'm wondering if others who are less experienced with "hardcore programming" might feel the same way?

Reply:
Yes it is about code/software. We do not talk about hardware programming (whatever that means) but someone who spends the majority of her/his time to write codes (a script is a code).

Maybe we should use computer programs (https://en.wikipedia.org/wiki/Computer_program)?

Reply:
It might be me who misunderstood which kind of contributions we are looking for. I imagined that it could be any kind of project/work/contribution related to "building the community", "the RSE initiative/role", FAIR and open research, to more technical things like documentation, work-flow management, scripting and coding. Maybe someone is working on setting up a journal, podcast, or whatever project, for researchers/RSEs. Would that be interesting?

Reply:
>We do not talk about hardware programming

that is a typo and you meant hardcore (from patric comment), right? Excluding people who do e.g. FPGA programming is very unfair imho, especially since in the end of the day dedicated hardware is the ultimate solution?

Reply:
Yes! Not "hardware" but "hardcore"! my mobile phone likes to "correct" me!

About having project/work/contribution related to "building the community": I think for once we should focus on those who spend most of their time writing codes (and not necessarily "big" codes). They are still hidden to everyone, including funders, decision makers, etc.
No offense but I see a lot of great talks/discussions around RSEs but few talks by people who effectively write codes most of their time. However, most scientific results/papers rely on software/code/programs/scripts, etc.  Papers in the Nordics are no exception. If I were a funder, I would conclude that there is no RSEs in the Nordics and obviously no need to have any. Let's show that there are RSEs in the Nordics: many are isolated, not recognized, managed by incompetents (related to RSE work) and very often they have non-permanent positions.
It may be less shiny for a (un)conference but I don't really see how we can build a community of RSEs with no-one writing codes.

Reply:
I'm not opposed to having a large focus on code as you suggest, and I like your idea of lifting out these roles. I guess I'm just worried about excluding certain contributions if our application reads as "code only". I don't see why one group should have to exclude the other. Then again, I have not attended a lot of these events, so if coders are underrepresented, then I completely trust your word on it.

Reply:
I agree with all the above but also somebody who is not writing code yet but is thinking about starting soon/later, is also welcome (?)